uktrade/data-workspace-frontend
# Feat/dt 2027 self certify validation

### Description of change

Ticket: [DT-2027](https://uktrade.atlassian.net/jira/software/projects/DT/boards/356?isInsightsOpen=true&selectedIssue=DT-2027)

This change adds validation to the self certify form.

### How to test
1. Visit `/request-access/self-certify` 

Validation errors should occur when you:

1. Try to add a date thats in the future
2. Try to add a date thats over a year old
3. Try to add an empty date
4. Try to add an incorrect date
5. Try to submit without accepting the declaration

### Checklist
